### DISCUSSION TITLE
[Question - ILI9341]: TFT_off vs. TFT_wake

### DISCUSSION THREAD

**Discussion by hasenradball** [Bodmer/TFT_eSPI#1781] [Q&A - General]
Hi,
whats the technical difference between `TFT_off()` and `TFT_sleep()`?
```
 void TFTweather::TFT_sleep() {
    // bring display into sleep
    tft.writecommand(0x10);
    // Delay for shutdown time before another command can be sent
    delay(5);
  }

  void TFTweather::TFT_off() {
    // switch off display
    tft.writecommand(0x28);
    // Delay for power supplies to stabilise
    delay(120);
  }

```
Is there any technical difference?

**Comment by Bodmer** [comment]
It will tell you in the datasheet. ISTR the tft.writecommand(0x10); shuts down the internal power to a minimum.  tft.writecommand(0x28); simply blanks the display.

**Comment by hasenradball** [comment]
Thanks for the reply.
But the Back LED has to switched off separately right?
